client/src/components/CreateMeetup/CreateMeetup.js

```diff
@@ -68,6 +85,44 @@ function CreateMeetup() {
     "cooking",
   ];
 
+  useEffect(async () => {
+    // TODO: redirect to login page for unauthenticated users.
+    // - Redux store may need a 'logging in' action to distinguish between unauthenticated
+    //   users and authenticated users for whom userID is not loaded in store yet.
+    if (!id || !userID) {
+      return;
+    }
+    const { res, resJSON } = await fetcher.getMeetup(id);
+    // if meetup does not exist (or error fetching meetup), redirect to /create
+    if (!res.ok) {
+      history.replace(`/create`);
+      return;
+    }
+    // if user is not meetup owner, redirect to /create
```

Comment:
Not sure if yall discussed this but wouldn't it make more sense to redirect to /meetup/:id instead of /create?

Reply:
Hm yes that does make more sense. I'll change that.